test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
> Is the problem that your JSON-schema declares an enum of numbers (rather than an enum of strings with corresponding tsEnumValues) for compatibility with jsonschema2pojo?

Exactly.  But I think more importantly, the point of json-schema is to declare valid values for what goes over the wire, and in this case for an integer-based enum it will be integers, which I think implies that `enum` should contain the integers.  What do you think?